tools/fapi/tss2_template.c

```diff
@@ -374,7 +374,13 @@ int main(int argc, char *argv[]) {
 
 int open_write_and_close(const char* path, bool overwrite, const void *output,
     size_t output_len) {
-    size_t length = output_len ? output_len : strlen (output);
+
+    size_t length = 0;
+
+    if (output_len){
```

Comment:
space between closing parenthesis and opening brace.

Reply:
Addressed in the latest commit.